src/styles/global.css

```
@@ -88,3 +88,67 @@ span[id="footnote-label"] {
   border-bottom: 1px solid theme(colors.muted.foreground / 40%);
   font-weight: 700;
 }
+
+@font-face {
+  font-family: "LINE Seed JP";
+  src: url("/fonts/line-seed-jp/LINESeedJP_OTF_Rg.woff2") format("woff2");
+  font-weight: regular;
```

```suggestion
  font-weight: theme(fontWeight.normal);
```

https://developer.mozilla.org/ja/docs/Web/CSS/font-weight#%E4%B8%80%E8%88%AC%E7%9A%84%E3%81%AA%E5%A4%AA%E3%81%95%E5%90%8D%E3%81%A8%E3%81%AE%E5%AF%BE%E5%BF%9C